### PR TITLE
[zh] improve performance-and-scalability requirements

### PR DESCRIPTION
**Pull request by my-git9**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/ops/deployment/performance-and-scalability/index.md [hunk 1]
```diff
@@ -21,13 +21,18 @@ owner: istio/wg-environments-maintainers
...
 
-Istio 的数据平面组件 Envoy 代理用来处理通过系统的数据流。控制平面组件如 Pilot、Galley 和 Citadel 负责配置数据平面。数据平面和控制平面有不同的性能关注点。
+Istio 的数据平面组件 Envoy 代理用来处理通过系统的数据流。控制平面组件如
+Pilot、Galley 和 Citadel 负责配置数据平面。数据平面和控制平面有不同的性能关注点。
 
 ## Istio {{< istio_release_name >}} 性能总结 {#performance-summary-for-Istio}
 
-[Istio 负载测试](https://github.com/istio/tools/tree/master/perf/load)网格包含了 **1000** 个服务和 **2000** 个 sidecar，全网格范围内，QPS 为 70,000。
+[Istio 负载测试](https://github.com/istio/tools/tree/master/perf/load)网格包含了
+**1000** 个服务和 **2000** 个 sidecar，全网格范围内，QPS 为 70,000。
```
Comment (wilsonwu) [inline content/zh/docs/ops/deployment/performance-and-scalability/index.md:35]:
```suggestion
**1000** 个服务和 **2000** 个 Sidecar，全网格范围内，QPS 为 70,000。
```

File: content/zh/docs/ops/deployment/performance-and-scalability/index.md [hunk 2]
```diff
@@ -105,9 +127,10 @@ the two proxies add about 1.7 ms and 2.7 ms to the 90th and 99th percentile late
 We obtained these results using the [Istio benchmarks](https://github.com/istio/tools/tree/{{< source_branch_name >}}/perf/benchmark)
 for the `http/1.1` protocol, with a 1 kB payload at 1000 requests per second using 16 client connections, 2 proxy workers and mutual TLS enabled.
 -->
-在 mesh 内部，请求会依次遍历客户端和服务器端代理。在 Istio {{< istio_release_name >}} 的默认配置中（即带有遥测 v2 的 Istio），
-两个代理分别在基线数据平面延迟的 90 和 99 分位延迟上增加约 1.7 和 2.7 毫秒。
-我们使用 `http/1.1` 协议的 [Istio 基准测试](https://github.com/istio/tools/tree/{{< source_branch_name >}}/perf/benchmark)获得了这些结果，
+在 mesh 内部，请求会依次遍历客户端和服务器端代理。在 Istio {{< istio_release_name >}}
```
Comment (wilsonwu) [inline content/zh/docs/ops/deployment/performance-and-scalability/index.md:130]:
```suggestion
在网格内部，请求会依次遍历客户端和服务器端代理。在 Istio {{< istio_release_name >}}
```

**Review by hanxiaop — COMMENTED** [review]
这篇在英文界面暂时已经没有了，FYI https://github.com/istio/istio.io/pull/12650

**Comment by windsonsea** [comment]
Hi @hanxiaop, the filename `index.md.disable` is applied now. Seems it hurts nothing if merged. How do you think?

@my-git9 If a file is marked with `disable`, it might not be required to maintain it again in my understanding.

**Comment by hanxiaop** [comment]
> Hi @hanxiaop, the filename `index.md.disable` is applied now. Seems it hurts nothing if merged. How do you think?
> 
> Hi @my-git9, if a file is marked with `disable`, it might not be required to maintain it again in my understanding.

IMO we should switch it to disable - which is the same in the EN doc site. To revise/not revise the content is not a big concern since this will eventually be changed again once the EN page come back.

**Review by windsonsea — APPROVED** [review]
Let's kick this in. It's consistent with en now.

**Pull request merged**